test/NuGet.Core.Tests/NuGet.Packaging.Test/PackageBuilderTest.cs

```diff
@@ -81,7 +81,6 @@ public void CreatePackageWithEmptyFoldersForV3Folders()
                     var files = archive.Entries
                         .Where(file => file.Name == "_._")
                         .Select(file => file.FullName)
-                        .OrderBy(s => s)
```

Comment:
This test `CreatePackageWithEmptyFoldersForV3Folders`, and `CreatePackageWithDifferentFileKinds` are both failing, but only on Windows. I don't have any immediate ideas why.

Reply:
lol, you're right.  `(int)'/'` has value 47, whereas `\` is 92. `A` through `Z` are 65 though 90.